docs/crr341.adoc

```diff
@@ -150,15 +150,35 @@ A design principle of SONAR-netCDF4 has been to focus on describing the acoustic
 
 NetCDF4 has two main organizational concepts: (i) the variable, which can contain a variety of data structures, and (ii) groups, being a collection of variables. Groups and variables can have attributes attached to them. Groups can be arranged into a hierarchy.
 
-SONAR-netCDF4 divides sonar data into seven netCDF4 groups:
+[plantuml, format=svg, opts="inline"]
+----
+legend
+Top-level
+|_ Annotation
+|_ Environment
+|_ Platform
+|_ Provenance
+|_ Sonar
+  |_ Beam_group
```

Comment:
The formal name for beam groups has a trailing number, so to be complete I suggest the list have something like Beam_group1, ..., Beam_groupN. For multi-channel echosounders there is a Beam_groupX for each channel.

Suggest to use 'Gridded' instead of Grid_group (the group in the Beam_group means a group of beams, not a netCDF group). The same numbering thing can apply to Gridded to allow for multiple grids at different resolutions, with the difference that a Gridded can contain data from multiple Beam_groups.

For the Interpretation group, the current proposal is to have groups under it for multiple versions of an interpretation. These would be called v1, v2, ..., vN. This is to allow for multiple revisions of an interpretation. Completely interpretations could be stored under Interpretation1, ..., InterpretationN. There will need to be link to indicate which Beam_groupXs and GriddedXs an Interpretation applies to. This is beginning to sound a bit complex, so alternate suggestions are welcome.

Reply:
Commit, 0aec092 address this comment.